internal/server/statvar/statvar_group.go

```diff
@@ -160,18 +161,35 @@ func GetStatVarGroup(
 		}
 		// Loop through import group by order. The stat var group is preferred from
 		// a higher ranked import group.
+		var customRootNode *pb.StatVarGroupNode
 		for _, btData := range btDataList {
 			for _, row := range btData {
 				svgData, ok := row.Data.(*pb.StatVarGroups)
 				if ok && len(svgData.StatVarGroups) > 0 {
 					for k, v := range svgData.StatVarGroups {
+						if k == customSvgRoot {
+							if customRootNode != nil {
+								continue
+							}
+							customRootNode = v
+						}
 						if _, ok := result.StatVarGroups[k]; !ok {
 							result.StatVarGroups[k] = v
 						}
 					}
 				}
 			}
 		}
+		if customRootNode != nil {
+			result.StatVarGroups[SvgRoot].ChildStatVarGroups = append(
+				result.StatVarGroups[SvgRoot].ChildStatVarGroups,
+				&pb.StatVarGroupNode_ChildSVG{
+					Id:                customSvgRoot,
+					SpecializedEntity: customRootNode.AbsoluteName,
+				},
+			)
```

Comment:
Just to confirm, should we check that the SvgRoot does not already have customSvgRoot?  For example, in old Stanford custom DC if schema was part of custom DC that can happen.

Reply:
That's good to know. Added a check.